### DISCUSSION TITLE
Jetson Orin Nano 4GB (P3767-0004) Ability to integrate mender to do rootfs A/B ?

### DISCUSSION THREAD

**Discussion by Links17** [OE4T/tegra-demo-distro#272] [Q&A]
Jetson Orin Nano 4GB (P3767-0004) Ability to integrate mender to do rootfs A/B

**Comment by dwalkes** [comment]
See discussion at [this thread](https://matrix.to/#/!YBfWVpJwNVtkmqVCPS:gitter.im/$dXBEBVuwsKb7LjqRutFYcqpKcohWAeP95p-ONwZTZ1o?via=gitter.im&via=matrix.org&via=3dvisionlabs.com) for latest status.

**Comment by Links17** [comment]
I'm so sorry... I'm still having a hard time building from Discussion, how can I build P3767-0004 mender from tegra-demo-distro kirkstone branch please?

**Reply by dwalkes** [reply]
Let's keep/move the discussion to element in order to minimize the number of cross threads.  See thread [here](https://matrix.to/#/!YBfWVpJwNVtkmqVCPS:gitter.im/$XW4pcCed4ZX6bEj5Q7cQSIbBxcauDiyBSSMU93NpmMc?via=gitter.im&via=matrix.org&via=3dvisionlabs.com).  Short answer is that there is no turn-key example or set of instructions you can follow available yet, unfortunately.
